content/modules/redisearch/_index.md

```diff
@@ -7,145 +7,67 @@ categories: ["Modules"]
+Sorting can be done by a given field as well and can be limited with an offset for simplified support for results pages.
+
+RediSearch is language aware and support the following languages for stemming:
+
+    "arabic", "danish", "dutch", "english", "finnish", "french", "german", "hungarian", "italian", "norwegian", "portuguese", "romanian", "russian", "spanish", "swedish", "tamil", "turkish"
+
+The index supports auto-complete engines with specific commands that can provide real-time interactive search suggestions.
 
 ## RediSearch Index
 
-The fundamental part of any search engine is what's called an Inverted
-Index. In very simple terms, a search index is a map between words or
-terms, to the respective documents they appear in.
-
-If we have two documents, one titled _Hello World_ and the other titled
-"Hello Kitty", a search engine would keep an Inverted Index of the word
-"hello" that contains two records, one for each of the aforementioned
-documents.
-
-Searching consists of loading, traversing, intersecting (if needed) and
-sorting these indexes in order to produce the most relevant results. It
-gets much more complicated, of course, as those indexes contain relevant
-scores, term positions in the document and more - but that's beyond the
-scope of this document.
-
-Until now, Inverted Indexes on top of a Redis database were always
-modeled with Redis native data types, usually Sorted Sets, where the
-document ID was stored as the "element" and the relevance as the
-"score." This worked great because it allowed developers to use Redis
-intersection and union to perform multi-word searches, but the approach
-has its limitations: it doesn't support exact phrase search, it has a
-big memory overhead, and can be very slow with big records
-intersections.
+The fundamental part of any search engine is the inverted index.
+The search index is a map between words or terms to the respective documents they appear in.
+
+If we have two documents, one titled _Hello World_ and the other titled "Hello Kitty",
+a search engine would keep an inverted index of the word "hello" that contains two records, one for each of these documents.
+
+Searching consists of loading, traversing, intersecting (if needed) and sorting these indexes in order to produce the most relevant results.
+The index gets more complex as the indexes contain relevant scores, term positions in the document, and other fine details.
+
+Until now, inverted indexes on top of a Redis database were always modeled with Redis native data types, usually sorted sets,
+where the document ID was stored as the "element" and the relevance as the "score."
+This worked great because it allowed developers to use Redis intersection and union to perform multi-word searches,
+but this approach doesn't support exact phrase search, it has a high memory overhead, and can be slow with big record intersections.
```

Comment:
This seems to trail off. If we want to continue this discussion of the implementation, then we need to talk about what makes RediSearch 2.0 different and be explicit about it.